src/invidious.cr

```diff
@@ -111,6 +116,11 @@ end
 OUTPUT = CONFIG.output.upcase == "STDOUT" ? STDOUT : File.open(CONFIG.output, mode: "a")
 LOGGER = Invidious::LogHandler.new(OUTPUT, CONFIG.log_level)
 
+# Run migrations
+if Invidious::Database::Migrator.new(PG_DB).pending_migrations?
+  puts "There are pending migrations. Run `invidious --migrate` to apply the migrations."
+  exit 46
```

Comment:
You said use a specific error code 😛. But for real... what error code should I return?

Reply:
I don't know xD I'll have to double check that, to ensure that we don't use something that either kemal or crystal already do.